syntax = "proto3";

package bakdata.corporate.v1;

message Corporate {
  string id = 1;
  uint32 rb_id = 2;
  string state = 3;
  string reference_id = 4;
  string name = 5;
  string event_date = 6;
  string event_type = 7;
  Status status = 8;
  string address = 9;
  string industry_description = 10;
  string capital = 11;
  repeated string ceo = 12;
  string information = 13;
}

enum Status {
  STATUS_UNSPECIFIED = 0;
  STATUS_INACTIVE = 1;
  STATUS_ACTIVE = 2;
}
